src/main/java/org/prebid/server/bidder/huaweiads/HuaweiAdSlotBuilder.java

```diff
@@ -141,13 +147,80 @@ private List<Asset> parseNativeRequestAssets(Native xNative) {
     private static List<String> makeDetailedCreativeTypeList(long numVideo, long numImage) {
         if (numVideo >= 1) {
             return List.of("903");
-        } else if (numImage > 1) {
-            return List.of("904");
-        } else if (numImage == 1) {
-            return List.of("901");
+        } else if (numImage >= 1) {
+            return List.of("901", "904", "905");
         } else {
             return List.of("913", "914");
```

Comment:
this `else` case is not the case anymore

also, these type should be combined if `numVideo >= 1 && numImage >=1`

Reply:
`numVideo >= 1 && numImage >=1`
Could please, explain where you found this? I can't find this in `huaweiads.go` file

Reply:
I see... Will do, NP.